src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -128,6 +129,12 @@ namespace
             // Apply turbidity bias.
             m_values.m_turbidity += BaseTurbidity;
 
+            const Source* size_multiplier_src = get_inputs().source("size_multiplier");
+            if (size_multiplier_src && size_multiplier_src->is_uniform())
```

Comment:
We should probably even emit a warning if the size multiplier is not uniform (uniform = constant) since it wouldn't make much sense.